### DISCUSSION TITLE
Pull shark is not awarded for all old PR's

### DISCUSSION THREAD

**Discussion by Likqez** [community/community#18539] [Other Feature Feedback, Questions, & Ideas] [answered]
I received 2x Pull Shark achievements for repositories from 2020. But the PR's merged this year are missing somehow.

**Comment by Belikeoscar** [comment]
I noticed that too, I got mine for a PR dating back to early 2021 but it doesn't recognize my recent PR.

**Comment by Exortions** [comment]
Was just about to make a discussion about this, I have pull requests that were merged many months ago that show up on my profile, but I have multiple pull requests in the past few months that don't show up on my profile.

**Comment by Likqez** [comment, marked as answer]
Got it, only the milestones show up in the history.
1st, 2nd, 16th, 128th and 1024th
